daos-cortx/docs/datamovment_with_s3.md

```diff
@@ -0,0 +1,139 @@
+# Daos object movement using aws s3cli
+
+- Daos source container to Daos destination container object movement is possible using aws s3 cli. This documents is the setup guide to perform mentioned exercise.
+
+# Test setup
+
+## Prerequisites
+
+* Single node deployment of cortx on another VM
+
+    - The single node deployment for CORTX will hosting s3 server, which will be used to manage buckets and store objects requested aws s3 cli on daos node.
```

Comment:
The single node deployment for CORTX hosting s3 server, which will be used to manage buckets and store objects moved from DOAS using aws s3 cli.

Reply:
done